src/hooks/__tests__/useInterval.spec.ts

```diff
@@ -0,0 +1,100 @@
+import { renderHook, act } from '@testing-library/react'
+import { useInterval } from '../use-interval'
+import { describe, it, expect, vi, beforeEach, afterEach } from 'vitest'
+
+describe('useInterval', () => {
+  beforeEach(() => {
+    vi.useFakeTimers()
+  })
+
+  afterEach(() => {
+    vi.restoreAllMocks()
+  })
+
+  it('should call the callback function at the specified interval', () => {
```

Comment:
This test suite does not test the `startInterval` but it mocks the function as we see, there is no call for `startInterval` to make sure that this function is called once time.